bin/chat_server_log.log

```diff
@@ -0,0 +1,4192 @@
+Apr 04, 2017 5:37:42 PM codeu.chat.ServerMain main
```

Comment:
Probably a good idea not to submit log files to source control. You can add the file name to .gitignore to prevent this.